java/gandiva/src/main/java/org/apache/arrow/gandiva/evaluator/JavaSecondaryCacheInterface.java

```diff
@@ -36,9 +36,11 @@ public BufferResult(long address, long size) {
 
   }
 
-  BufferResult getSecondaryCache(long addr, long size);
+  BufferResult get(long addr, long size);
 
-  void setSecondaryCache(long addrExpr, long sizeExpr, long addr, long size);
+  void set(long addrExpr, long sizeExpr, long addr, long size);
```

Comment:
Can you add comments  before the methods that also describes the parameters?
Also rename the parameters to more clear names..like keyBufAddr, keyBufSize, valueBufAddr, valueBufSize